wd_core.au3

```diff
@@ -262,12 +269,14 @@ Global $_WD_ERROR_OUTPUTDEBUG = False      ; Log errors to "OutputDebugString"
 Global $_WD_DEBUG = $_WD_DEBUG_Info        ; Trace to console and show web driver app
 Global $_WD_CONSOLE = ConsoleWrite         ; Destination for console output
 Global $_WD_CONSOLE_Suffix = @CRLF         ; Suffix added to the end of Message in $_WD_CONSOLE function
+Global $_WD_IFILTER = 16                   ; Passed to _HtmlTableGetWriteToArray to control filtering
```

Comment:
This was removed in the previous release

Reply:
oops

Reply:
take a look at:
[$_WD_IFILTER removal](https://github.com/Danp2/au3WebDriver/pull/492/commits/01898c06db99a0d9be92bf1b5d247580aaa05081)